foundation_cms/static/scss/utilities.scss

```diff
@@ -0,0 +1,9 @@
+.grid-container {
+  // this class is useful if you need to have a full-bleed grid-container on mobile
+  &.mobile-padding-0 {
```

Comment:
not used in this PR yet but will be used in block template code once this PR is merged